temporalcli/commands.workflow_trace.go

```diff
@@ -0,0 +1,67 @@
+package temporalcli
+
+import (
+	"fmt"
+	"github.com/fatih/color"
+	"github.com/temporalio/cli/temporalcli/internal/printer"
+	"github.com/temporalio/cli/temporalcli/internal/trace"
+	"go.temporal.io/sdk/client"
+)
+
+func (c *TemporalWorkflowTraceCommand) run(cctx *CommandContext, _ []string) error {
+	cl, err := c.Parent.ClientOptions.dialClient(cctx)
+	if err != nil {
+		return err
+	}
+	defer cl.Close()
+
+	ctx := cctx.Context
```

Comment:
This is not needed, `cctx` satisfies the context interface by embedding context, so it can be used directly as a context